app/models/deleted_object.rb

```diff
@@ -8,7 +8,7 @@ class DeletedObject < ApplicationRecord
     scope scoped_class.to_s.underscore.pluralize.to_sym, -> { where(object_type: scoped_class) }
   end
 
-  scope(:deleted_owner, lambda do
+  scope(:deleted_owner, -> do
```

Comment:
Perhaps now we can do: `scope :deleted_owner, -> do`
And then in like 18: `end`